cmd/tendermint/commands/flags/log_level.go

```diff
@@ -0,0 +1,85 @@
+package flags
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	cfg "github.com/tendermint/tendermint/config"
+	"github.com/tendermint/tmlibs/log"
+)
+
+const (
+	defaultLogLevelKey = "*"
+)
+
+// IsLogLevelSimple returns true if log level is a single word ("info", ...).
+func IsLogLevelSimple(l string) bool {
```

Comment:
Can we just expose one method that Parses simple or complex log levels?

We can keep these two, just add one that does the check for simple, like you do in root.go